src/openms/include/OpenMS/QC/QCBase.h

```diff
@@ -52,6 +53,44 @@ namespace OpenMS
   class OPENMS_DLLAPI QCBase
   {
   public:
+
+    /// two way mapping from ms-run-path to protID|pepID-identifier
+    struct Mapping
```

Comment:
sounds useful for other parts in OpenMS as well

Reply:
true. suggestion? new class? a bit of a hassle for this little thing, but ok.

Reply:
Maybe in protein Id ?